crime-assessment-service/src/main/java/uk/gov/justice/laa/crime/assessmentservice/iojappeal/validator/ApiCreateIojAppealRequestValidator.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.laa.crime.assessmentservice.iojappeal.validator;
+
+import lombok.experimental.UtilityClass;
+import uk.gov.justice.laa.crime.common.model.ioj.ApiCreateIojAppealRequest;
+import uk.gov.justice.laa.crime.common.model.ioj.IojAppeal;
+import uk.gov.justice.laa.crime.common.model.ioj.IojAppealMetadata;
+import uk.gov.justice.laa.crime.enums.IojAppealAssessor;
+import uk.gov.justice.laa.crime.enums.NewWorkReason;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@UtilityClass
+public class ApiCreateIojAppealRequestValidator {
+    private static final String FIELD_IS_MISSING = "%s is missing.";
+
+    public List<String> validateRequest(ApiCreateIojAppealRequest request) {
+        var errorList = new ArrayList<String>();
+
+        // check the IojAppealMetadata Object.
+        errorList.addAll(validateMetaData(request));
+        // check the IojAppeal Object.
+        errorList.addAll(validateIoJAppeal(request));
+
+        return errorList;
+    }
+
+    // MetaData Validation
+
+    List<String> validateMetaData(ApiCreateIojAppealRequest request) {
+        List<String> errorList = new ArrayList<>();
+        IojAppealMetadata metadata = request.getIojAppealMetadata();
```

Comment:
As we are only validating the metadata within this method would it be a bit cleaner to just pass that object in rather than the request? (same goes for the validateAppeal method).